test_debug_mode.py

```
@@ -0,0 +1,93 @@
+    
+    # Force reload modules to apply new environment
+    import sys
+    for mod in list(sys.modules.keys()):
+        if mod.startswith('praisonaiagents'):
+            del sys.modules[mod]
+    
+    import warnings
+    warnings.simplefilter('always')
+    
+    try:
+        import praisonaiagents
```

Ruff (0.12.2)

`praisonaiagents` imported but unused

Remove unused import: `praisonaiagents`

(F401)